cmd/dashboard.go

```diff
@@ -27,10 +26,14 @@ const (
 	// defaultLocalPort is the default local port used for port forwarding for `dapr dashboard`
 	defaultLocalPort = 8080
 
+	// defaultNamespace is the default namespace where the dashboard is deployed
+	defaultNamespace = "dapr-system"
```

Comment:
rename this var to daprSystemNamespace and introduce another variable defaultNamespace for "default"